### PR TITLE
Update JSON schema to include `command.variables`

### PR DESCRIPTION
**Pull request by DannyBen**
Update schema for #563

### CODE REVIEW

**Comment by DannyBen** [comment]
@EmilyGraceSeville7cf, I have added a new possible value for `command` called `variables`.

It is an array of dictionaries, and each item must have a `name` and an optional `value`. The `value` can be pretty much anything:

```yaml
    variables:
    # Simple value
    - name: output_folder
      value: output

    # Array
    - name: download_sources
      value:
      - youtube
      - instagram

    # Associative array
    - name: zip_options
      value:
        pattern: "*.json"
        compression_level: fast
```

So I did not limit the value to any particular type.
Note that bashly.json schema is auto-generated from the bashly.yaml schema.

Your approval por favor.

File: support/schema/bashly.yml [hunk 1]
```diff
@@ -435,6 +460,15 @@ definitions:
     uniqueItems: true
     items:
       $ref: '#/definitions/environment-variable'
+  variables-property:
+    title: variables
+    description: |-
+      Bash variables for the current application
+      https://bashly.dannyb.co/configuration/command/#variables
+    type: array
+    uniqueItems: true
```
Comment (EmilyGraceSeville7cf) [inline support/schema/bashly.yml:469]: I would also add `minItems: 1`.

File: support/schema/bashly.yml [hunk 2]
```diff
@@ -335,6 +335,31 @@ definitions:
...
+        title: name
+        description: A name for the current variable
+        type: string
+        minLength: 1
+        examples:
+          - output_dir
+      value:
+        title: value
+        description: A value for the current variable
+        examples:
```
Comment (EmilyGraceSeville7cf) [inline support/schema/bashly.yml:357]: What if the variable value is `null` ? Is it an error? What it corresponds to in Bash (in the context of Bashly) if it's not?

**Review by EmilyGraceSeville7cf — APPROVED** [review]
LGTM, just have some questions and comments 😄

**Comment by DannyBen** [comment]
Thanks @EmilyGraceSeville7cf - merging.

**Pull request merged**